search/views_test.py

```diff
@@ -120,6 +127,49 @@ def test_staff_and_instructor_can_access(self, role):
         self.client.force_login(user)
         self.assert_status_code()
 
+    @ddt.data(
+        Staff.ROLE_ID,
+        Instructor.ROLE_ID,
+    )
+    def test_staff_and_instructor_in_other_program_no_results(self, role):
```

Comment:
Can you combine this test with the next one and use a parameter for the existence of `ProgramEnrollment`?